src/stores/errors/errorsReducer.ts

```diff
@@ -0,0 +1,37 @@
+import * as Redux from 'redux';
+
+import { ErrorsActions } from './errorsActions';
+
+export interface ErrorsStoreState {
+  [errorKey: string]: unknown;
+}
+
+const initialState: ErrorsStoreState = {};
+
+export function errorsReducer(
+  state = initialState,
+  action: Redux.AnyAction,
+) {
+  switch (action.type) {
+    case ErrorsActions.SET: {
+      const error = action.payload;
+
+      return {
+        ...state,
+        ...error,
+      };
+    }
+
+    case ErrorsActions.RESET: {
+      const { errorKey } = action.payload;
+      const filteredState = Object.fromEntries(Object.entries(state).filter(key => key[0] === errorKey));
```

Comment:
instead, do the following:
```
delete state[errorKey]

return {
   ...state
}
```

Reply:
i think it's no working...

Reply:
this code is not working...
i think,we need :
const { errorKey } = action.payload;
const filteredState = Object.fromEntries(Object.entries(state).filter(key => key[0] === errorKey));

  return {
    ...filteredState,
  };